editor/src/clj/editor/gui.clj

```diff
@@ -837,7 +837,7 @@
 
 (defn- strip-size-from-node-desc [node-desc]
   (-> node-desc
-      (assoc :size [0.0 0.0 0.0 0.0])
+      (dissoc :size)
```

Comment:
Instead of explicitly setting the manual size to zero, we clear it out. This results in the property not being assigned when loading, so the default value of 200 x 100 pixels is used for the `manual-size` property.